### ISSUE TITLE
Videos are stretched and not cropped

### ISSUE THREAD

**Issue by breedx2** [breedx2/cuspid#50]
Would be nice to have aspect ratio remain the same, without stretching.  Can HTML5 video be cropped?

**Comment by zkarcher** [comment]
I'm not sure about cropping HTML5 video.

In ThreeJS, the texture coordinates could be moved towards the center, to draw a cropped region of the video texture. (I think the zoom/scale effect did this.)

**Comment by breedx2** [comment]
Yeah.  I got around this with the images by doing the cropping up-front right after load, which was really nice.  I see where you're coming from with the zoom thing -- and that seems like a good approach...but I have to keep in mind that generic zooming is already supported, so it would have to compensate for that.  Probably not crazy, but just more to think about.
